docs/section6/Rule6-13.md

```diff
@@ -38,41 +34,26 @@
 
         - Get matching space from B_RMR: `space_b = match_data_element(B_RMR, Spaces, space_p.id)`  
 
-          - Get normalized space lighting schedule for B_RMR: `normalized_schedule_b = normalize_space_lighting_schedule(space_b)`  
-
-        - Get lighting space type: `lighting_space_type_p = space_p.lighting_space_type`  
-
-          - If lighting space type is employee lunch and break rooms, conference/meeting rooms, or classrooms (not including shop classrooms, laboratory classrooms, and preschool through 12th-grade classrooms), get baseline lighting occupancy sensor reduction factor from Table G3.7: `if lighting_space_type_p in ["LOUNGE/BREAKROOM", "CONFERENCE/MEETING/MULTIPURPOSE ROOM", "CLASSROOM/LECTURE HALL/TRAINING ROOM - PENITENTIARY", "CLASSROOM/LECTURE HALL/TRAINING ROOM - ALL OTHER"]: reduction_factor_b = data_lookup(table_G3_7, lighting_space_type_p, "FULL-AUTO ON")`  
-
-          - Else, set baseline reduction factor to 0: `else: reduction_factor_b = 0`  
+          - Get normalized space lighting schedule for B_RMR: `normalized_schedule_b = normalize_space_schedules(space_b)`  
 
-        - For each lighting in space: `lighting_p in space_p.interior_lighting:`  
+        - Get normalized space lighting schedule: `normalized_schedule_b = normalize_space_schedules(space_p)`
 
-          - Get lighting control type: `lighting_control_type_p = lighting_p.occupancy_control_type`  
+        - Compare lighting schedules in P_RMR and B_RMR: `schedule_comparison_result = compare_schedules(normalized_schedule_b, normalized_schedule_b, building_open_schedule_p, adjusted_reduction_factor_p)`  
 
-          - Get lighting occupancy sensor reduction factor from Table G3.7: `reduction_factor_p = data_lookup(table_G3_7, lighting_space_type_p, lighting_control_type_p)`  
+          **Rule Assertion:**
 
-          - Calculate adjusted lighting occupancy sensor reduction factor based on B_RMR: `adjusted_reduction_factor_p = ( 1 - reduction_factor_p ) / ( 1 - reduction_factor_b )`  
+          - Case 1: For all hours, for each lighting, if lighting schedule in P_RMR is equal to lighting schedule in B_RMR times adjusted lighting occupancy sensor reduction factor: `if schedule_comparison_result == "MATCH": PASS`  
 
-          - Get lighting schedule: `lighting_schedule_p = lighting_p.lighting_multiplier_schedule`  
+          - Case 2: Else if lighting schedule in P_RMR is lower than or equal to lighting schedule in B_RMR times adjusted lighting occupancy sensor reduction factor: `if schedule_comparison_result == "EQUAL AND LESS": CAUTION`  
 
```

Comment:
I think this should be a FAIL, because P-RMR schedule fractions is less than B-RMR schedule fraction times adjustment. Or am I missing what this is doing?

Reply:
It still shows as "CAUTION" for me...

Reply:
@mkarpman Case 2 was updated to FAIL. Not sure why you still see CAUTION.